c8/api.py

```diff
@@ -82,7 +81,7 @@ def context(self):
         """
         return self._executor.context
 
-    def _execute(self, request, response_handler):
+    def _execute(self, request, response_handler, isUserApi=False):
```

Comment:
Allow specifying the prefix instead of the boolean, it might come in handy later for other endpoints that way.